tabular/src/autogluon/tabular/models/knn/knn_rapids_model.py

```diff
@@ -18,10 +18,10 @@ class KNNRapidsModel(KNNModel):
     RAPIDS KNearestNeighbors model : https://rapids.ai/start.html
 
     NOTE: This code is experimental, it is recommend to not use this unless you are a developer.
-    This was tested on rapids-0.18 via:
+    This was tested on rapids-21.06 via:
 
-    conda create -n rapids-0.18 -c rapidsai -c nvidia -c conda-forge -c defaults rapids-blazing=0.18 python=3.7 cudatoolkit=10.1 -y
-    conda activate rapids-0.18
+    conda create -n rapids-21.06 -c rapidsai -c nvidia -c conda-forge rapids=21.06 python=3.8 cudatoolkit=11.2
```

Comment:
Will this work with python 3.7? We haven't fully migrated to 3.8.

Reply:
Yes, it will work with both python 3.7 and 3.8

Reply:
Also, is CUDA 11.2 min requirement?

Reply:
CUDA 11.0 is also supported - I used the RAPIDS Release Selector from https://rapids.ai/start.html to generate the command specific to my local setup